src/PropertyManager/Travely.PropertyManager.Service/Extensions/ExpressionHelper.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using Travely.PropertyManager.Service.Models.Base;
+
+namespace Travely.PropertyManager.Service.Extensions
+{
+    public static class ExpressionHelper
```

Comment:
Do we actually need this ?

Reply:
This utility class will save development time, I think it's a good idea to use it here.